prime-dotnet-webapi/Controllers/HealthAuthoritySitesController.cs

```diff
@@ -60,9 +60,9 @@ public async Task<ActionResult> CreateHealthAuthoritySite(int healthAuthorityId,
             {
                 return Forbid();
             }
-            if (!await _healthAuthorityService.VendorExistsOnHealthAuthorityAsync(healthAuthorityId, payload.HealthAuthorityVendorId))
+            if (!await _healthAuthorityService.VendorExistsOnHealthAuthorityAsync(healthAuthorityId, payload.HealthAuthorityCareTypeId))
```

Comment:
This seems to be calling the wrong method, that is, checking vendors with care type ID?

Reply:
Good catch. Not sure what I was thinking. The validation has been moved to Update/PUT mothed and revert back to passing VendorId.